README.md

```diff
@@ -0,0 +1,115 @@
+# DeltaKernel Change Impact Analysis Tool
+
+## Table of Content
+
+- [Introduction](#introduction)
+- [Innovation](#innovation)
+- [How to Use](#how-to-use)
```

Comment:
I recommend adding an `Assumptions` section that describes all of the assumptions you are making about the state of the user's Linux kernel source repository clone, how that clone will be used by your script, and what state it will be in when your script is complete. You may also want to provide some manual cleanup guidance in case something goes wrong and the user needs to "reset" themself to a known good configuration.

Reply:
I added sections inside the "How to Use" section explaining both how the tool runs and the stages of cloning the Linux repository after execution. I also added manual cleanup guidance and instructions on resetting the Linux repository in case of git conflicts.

Reply:
https://github.com/elisa-tech/delta-kernel/issues/2